content/en/observability_pipelines/monitoring/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Metrics
+disable_toc: false
+aliases:
+- /path-to-old-doc/
+further_reading:
+- link: "/metrics/summary/"
+  tag: "Documentation"
+  text: "Learn more about the Metrics Summary"
+- link: "/metrics/explorer/"
+  tag: "Documentation"
+  text: "Using the Metrics Explorer to explore and analyze your metrics"
+- link: "/getting_started/dashboards/"
+  tag: "Documentation"
+  text: "Getting started with dashboards"
+- link: "/getting_started/monitors/"
+  tag: "Documentation"
+  text: "Getting started with monitors"
+---
+
+## Overview
+
+Use Observability Pipelines' metrics to create [dashboards][1], [notebooks][2], and [monitors][3] for you specific use case. Use [Metrics Summary][5] to see metadata and tags available for the metrics. You can also see which dashboards, notebooks, monitors, and SLOs are using those metrics. See [Getting Started with Tags][4] for more information on how to use tags to group metrics by specific pipelines, Workers, and components.
```

Comment:
Might this make sense as a bulleted list? If not, or even if so, I think this needs an introductory sentence about this page itself.

Reply:
I added a sentence and make it in a bulleted list.